docs/template_plugin/src/template_infer_request.cpp

```diff
@@ -162,6 +145,66 @@ static void blobCopy(const Blob::Ptr& src, const Blob::Ptr& dst) {
                 }
             }
         } break;
+        case Precision::I64 : {
+            switch (dst->getTensorDesc().getPrecision()) {
+                case Precision::I64 : break;
+                case Precision::I32 : {
+                    blobCopy<int64_t , int32_t>(src, dst);
```

Comment:
we can use `ngraph::runtime::reference::convert` here

Reply:
good catch, fixed here https://github.com/openvinotoolkit/openvino/pull/5702